packages/next-on-pages/templates/_worker.js/routes-matcher.ts

```diff
@@ -79,6 +79,8 @@ export class RoutesMatcher {
 		this.wildcardMatch = wildcardConfig?.find(
 			w => w.domain === this.url.hostname,
 		);
+
+		this.locales = collectLocales(routes);
```

Comment:
btw, I think we should probably move this to build-time and inject them instead of doing it every time we create a new router. I lowkey have been thinking we should do so since I added the locale logic a few months back lol...

Reply:
yes that's a good point, we had something else like this and agreed on moving it in the build process, this could be moved there as well, I'll have a look 👍

Reply:
I've moved this to build time and introduced a new variable called `__BUILD_METADATA__` to pass this sort of data along, please have a look and let me know what you think 🙂👍

Reply:
perfect, thank you!